### PR TITLE
Create string-reverse.cpp

### PR DESCRIPTION
**Pull request by hazel11182**
Reversing a string

### CODE REVIEW

File: strings/c-or-cpp/string_reverse.cpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+#include<cstring>
```
Comment (Arsenic-ATG) [inline strings/c-or-cpp/string_reverse.cpp:2]: std::string is declared in `<string>` header and not in `<cstring>`.

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
on top of review comment left in the file, the [changes requested on your other PR](https://github.com/MakeContributions/DSA/pull/114#pullrequestreview-616053363) also imply here.

**Comment by ming-tsai** [comment]
Please check it again @Arsenic-ATG

**Comment by Arsenic-ATG** [comment]
@ming-tsai , she have just fixed one of the errors pointed out by me. 

she still have to add the program to index Readme.md and also change the naming convention to include hyphens"-" instead of underscores"_" in the name, as mentioned in contribution guidelines.

**Review by hazel11182 — COMMENTED** [review]
done. how do i add it to the readme index?

**Review by Arsenic-ATG — APPROVED** [review]

**Comment by ming-tsai** [comment]
Hi @hazel11182,
Nice work, thanks for your contributions, and welcome to our team.
> The invitation [link](https://github.com/MakeContributions)

**Pull request merged**